src/views/CharacterListView.js

```diff
@@ -27,9 +30,18 @@ class CharacterListView extends React.Component {
 
 // our mapStateToProps needs to have two properties inherited from state
 // the characters and the fetching boolean
+const mapStateToProps = state => {
+  console.log(state);
```

Comment:
Love seeing this `console.log()` here! Keep doing this. It's a great habit to have.